src/components/todo-filter.jsx

```diff
@@ -1,13 +1,37 @@
-import FilterLink from 'components/filter-link';
+import FilterRadio from 'components/filter-radio';
 import {allTodos, completedTodos, activeTodos} from 'redux/modules/filter-todos';
 
 const TodoFilter = () => (
-  <div>
-    Filter:
-    <FilterLink action={allTodos}>All</FilterLink>
-    <FilterLink action={completedTodos}>Completed</FilterLink>
-    <FilterLink action={activeTodos}>Active</FilterLink>
-  </div>
+  <fieldset className="filters">
+    <legend className="filters__title">Show: </legend>
+    <FilterRadio action={allTodos}>All</FilterRadio>
+    <FilterRadio action={completedTodos}>Completed</FilterRadio>
```

Comment:
I'm wondering if it makes sense to use a `ul` for the `FilterRadio` components? I'm not sure right now when you select one of the filter options how many components will be rerendered due to the change of the active class. Also generally it's not good practice to use input fields without a form tag that wraps them. I'm leaning more towards going back to filter links and using a `ul` to group and style them.

Reply:
I am for the radio buttons cause they fit the filters case better IMHO. Can you point some source why is not good to have only input fields without form - cause I think it is a fair usage.

Reply:
Have a look here for handling inputs in React. Pay special attention to the `Defining the Form and Its Events in React` paragraph.
https://www.sitepoint.com/work-with-forms-in-react/

And have a look at this documentation `Radio Button Input` paragraph
https://www.w3schools.com/html/html_forms.asp

Reply:
put the radio inputs in `form` tag, they already have name property - so should be fine now